test/integration/helpers/Npm.cs

```diff
@@ -0,0 +1,70 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Appium.Net.Integration.Tests.helpers
+{
+    internal class Npm
+    {
+        static private Platform envPlatform;
+
+        public static string GetNpmPrefixPath()
+        {
+            envPlatform = Platform.CurrentPlatform;
+            string npmPath = GetNpmExecutablePath();
+            string result = RunCommand(npmPath, "config get prefix");
+
+            result = result.Trim();
+
+            return result;
+        }
+
+        private static string RunCommand(string command, string arguments)
+        {
+            try
+            {
+                Process process = new Process
+                {
+                    StartInfo = new ProcessStartInfo
+                    {
+                        FileName = command,
+                        Arguments = arguments,
+                        RedirectStandardOutput = true,
+                        UseShellExecute = false,
+                        CreateNoWindow = true,
+                    }
+                };
+
+                process.Start();
+                string output = process.StandardOutput.ReadToEnd();
+                process.WaitForExit();
+
+                return output;
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine($"Error executing command: {ex.Message}");
+                return null;
```

Comment:
So Should I remove the entire Try-catch in your opinion?

Reply:
depending on what you want to achieve. 
This is the point of exceptions - they need to be thrown if there is an exceptional satiation in the workflow.

In this particular example I can propose three approaches:
- Let the code throw a generic exception
- Catch the exception and rethrow a more specific one, which describes the situation better for the end-user (e.g. instead of having ENOENT message show something like `Node Package Manager (npm) cannot be found. Make sure Node.js is installed and present in PATH`)
- Let the code throw a custom exception and catch it on a higher level to show a more specific error message to the user

In general I expect exceptions to be helpful and provide hints/workarounds on how to avoid them.

Reply:
Since we are in RunCommand() method, a specific exception for NPM might be tricky since if in the future we would like to use this for other cases we can't throw the above error

Reply:
on the other end, we are in npm class :)